### ISSUE TITLE
Can no longer build 

### ISSUE THREAD

**Issue by alatdneg** [PixarAnimationStudios/OpenTimelineIO#324]
Trying to "python ./setup.py build" after updating from 0.8 to current head.

error in OpenTimelineIO setup command: 'tests_require' must be a string or list of strings containing valid project/version requirement specifiers; Expected version spec in mock;python_version<"3.3" at ;python_version<"3.3"

cheers

**Comment by ssteinbach** [comment]
I suspect it might be that your setuptools module is out of date for some of the features put into the `setup.py`.

Even just to test, if you create a virtual env and update to the latest setup tools and pip, are you then able to build?
```
virtualenv otio_test
source otio_test/bin/activate
pip install pip --upgrade
pip install setuptools --upgrade
python setup.py build
```

There is a bigger question of whether we should be careful about which version of setuptools we're supporting.  I don't think the vfxplatform includes setuptools in its versioning.

**Comment by ssteinbach** [comment]
@alatdneg any luck with this?

**Comment by alatdneg** [comment]
I commented out the bit in the setup.py that broke it, working now..

**Comment by ssteinbach** [comment]
Before we close this, can you look up which version of setuptools you're using?  I think there is a way to guard against this, but it involves some hackery (we can put all arguments to setup in a dictionary and then add more of the optional fields depending on which version of setuptools is present in the host environment).

**Comment by ssteinbach** [comment]
Hopefully addressed by #397
